pkg/metrics/scheduler.go

```diff
@@ -200,6 +201,14 @@ func (m *SchedulerMetrics) AddAllocatedContainers(value int) {
 	m.allocatedContainers.Add(float64(value))
 }
 
+func (m *SchedulerMetrics) getAllocatedContainers() (value int, err error) {
```

Comment:
We should not mix named and unnamed returns.
We have standardised on unnamed everywhere else.

Reply:
Thanks, didn't know that. Is it standardised on a per file or a per package base?